src/qt/utilitydialog.cpp

```diff
@@ -28,9 +28,6 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
     QDialog(parent),
     ui(new Ui::HelpMessageDialog)
 {
-    ui->setupUi(this);
-    GUIUtil::restoreWindowGeometry("nHelpMessageDialogWindow", this->size(), this);
```

Comment:
I'm not sure it is terribly useful for the help message dialog to store it's position either.